tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "noImplicitAny": false,
```

Comment:
Since this is a young project, this would be a good option to set to `true`.

Other options that are nice: `noUnusedLocals`, `noImplicitReturns`

Reply:
nice I didn't know these options